src/components/nodesColumns/columns.tsx

```diff
@@ -111,6 +118,57 @@ export function getMemoryColumn<
         resizeMinWidth: 170,
     };
 }
+
+export function getRAMColumn<T extends {MemoryUsed?: string; MemoryLimit?: string}>(): Column<T> {
```

Comment:
There is no sorting for new columns if table is paginated.

Currently, for every column `name` is checked with `isSortableNodesProperty`, if `name` is in the list, the column is sortable and it's name is sent to backend as sort param.

I'm not sure what's to do in this case, since there are two columns with different ids, but they have the same sort fields (`CPU` and `Memory`). 

Probably, you can use not column `name`, but add separate `sortField` property to every column object (in such case it's better to do it in a separate PR, since it's a lot of refactoring). Maybe there is a better and easier solution

Reply:
created dictionary for sort values

not sure if it's the best one in the world, but as far as we decided to rename CPU to Pools looks like there are not so many choices left...